requirements.txt

```diff
@@ -0,0 +1,21 @@
+annotated-types==0.5.0
+anyio==3.7.1
+click==8.1.6
+exceptiongroup==1.1.3
+fastapi==0.101.1
+h11==0.14.0
+httptools==0.6.0
+idna==3.4
+pydantic==2.1.1
+pydantic-settings==2.0.3
+pydantic_core==2.4.0
+python-dotenv==1.0.0
+PyYAML==6.0.1
+slack-sdk==3.21.3
+sniffio==1.3.0
+starlette==0.27.0
+typing_extensions==4.7.1
+uvicorn==0.23.2
+uvloop==0.17.0
+watchfiles==0.19.0
+websockets==11.0.3
```

Comment:
Have you noticed anything we don't need?

Reply:
Actually I am wondering why do we need anything except `slack-sdk` and `fastapi` right now?

Reply:
cause slack_sdk and and fastapi use other packages that are installed as well.

Reply:
It would be good to separate it visually somehow so when we decide to delete some dependence we know what else should be deleted.

By the way, where did you get that list of what slack requires to install?

Reply:
requirements.txt is a copy of my environment I get by `pip freeze` command. When we decide to remove the package we will remove it from local environment and make a new copy of it into the file.